data/json/mapgen/bike_shop.json

```diff
@@ -175,30 +175,30 @@
     "object": {
       "fill_ter": "t_flat_roof",
       "rows": [
-        "                         ",
-        "                         ",
-        " |2222222222222222222223 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.....................3 ",
-        " |.............-------53 ",
-        " |......##;##..3         ",
-        " |.&....#___#..3         ",
-        " |.X....#_>_#=.3         ",
-        " -------#####---         ",
-        "                         "
+        "                        ",
+        "                        ",
+        " |222222222222222222223 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |....................3 ",
+        " |.............------53 ",
```

Comment:
Why not delete the spaces here? To match the building below?

Reply:
Yes, to match the ground floor.